packages/aws-cdk-lib/aws-apigatewayv2/lib/websocket/route.ts

```diff
@@ -76,11 +76,6 @@ export class WebSocketRoute extends Resource implements IWebSocketRoute {
   public readonly webSocketApi: IWebSocketApi;
   public readonly routeKey: string;
 
-  /**
-   * Integration response ID
-   */
-  public readonly integrationResponseId?: string;
-
```

Comment:
I assume this was added in anticipation of an implementation of integration responses, but serves no purpose and cannot store multiple responses